lib/inferno/apps/cli/templates/%library_name%.gemspec.tt

```diff
@@ -1,11 +1,15 @@
+require_relative 'lib/<%= library_name %>/version'
+
 Gem::Specification.new do |spec|
   spec.name          = '<%= library_name %>'
-  spec.version       = '0.0.1'
+  spec.version       = <%= module_name %>::VERSION
   spec.authors       = <%= authors %>
   # spec.email         = ['TODO']
   spec.date          = Time.now.utc.strftime('%Y-%m-%d')
-  spec.summary       = '<%= title_name %> Test Kit'
-  spec.description   = '<%= human_name %> Inferno test kit for FHIR'
+  spec.summary       = '<%= title_name %>'
+  # spec.description   = <<~DESCRIPTION
+  #   This is a big markdown description of the test kit.
+  # DESCRIPTION
```

Comment:
Are we intentionally keeping `MyTestKit::Metadata.description` separate from gemspec description?

Reply:
Yes, we can't rely on the metadata in the gemspec because then the inferno gem would have to be loaded in order to parse the gemspec.